scripts/src/errors.ts

```diff
@@ -291,3 +294,11 @@ export function TooManyRegistrations(message: string): MarketplaceError {
 export function TooMuchEarnOrdered(message: string): MarketplaceError {
 	return TooManyRequests(CODES.TooManyRequests.Amounts, message);
 }
+
+function DeprecationError(index: number, message: string): MarketplaceError {
+	return new MarketplaceError(410, index, "Request to a deprecated resource", message);
```

Comment:
why are we using 410 and not costs?

Reply:
`consts`? This is the same throughout the file. I don't mind changing it